types/wordpress__data-controls/index.d.ts

```diff
@@ -53,13 +52,12 @@ export function apiFetch(options: APIFetchOptions): unknown;
  * @returns An object for registering the default controls with the store.
  */
 export const controls: {
-    API_FETCH: (action: Action) => Promise<any>;
-    DISPATCH: (action: Action) => void;
-    SELECT: (action: Action) => any;
```

Comment:
These are no longer provided per the source code: https://github.com/WordPress/gutenberg/blob/721cb342060a5f6ddf34f25e5379e6d8a4c806ae/packages/data-controls/src/index.js